physical/raft/raft.go

```diff
@@ -1014,6 +1014,11 @@ func (b *RaftBackend) GetConfigurationOffline() (*RaftConfigurationResponse, err
 	config := &RaftConfigurationResponse{
 		Index: state.Index,
 	}
+
+	if config.Servers == nil {
+		return config, nil
```

Comment:
can you explain this? is no servers not an error?

Reply:
Nope, because this is essentially a cache of the previous raft state that the fsm records. If vault was never run for example, there'd be no servers. Diagnose will throw a warning downstream where it says `even number of servers detected: 0`, which I'm hoping will be a red flag to operators that have been running a vault server, but won't be as strong a nuisance as an error to those that have a fresh binary.